### PR TITLE
docs: fix parameter format in tidb_http_api.md

### PR DESCRIPTION
**Pull request by shawn0915**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

related this topic: https://asktug.com/t/topic/1033423/4
@zimulala could you help take a look?

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: docs/tidb_http_api.md [hunk 1]
```diff
@@ -582,7 +582,7 @@ timezone.*
 27. Get count {number} TiDB DDL job history information, start with job {id}
 
      ```shell
-     curl http://{TIDBIP}:10080/ddl/history?start_job_id={id}&limit={number}
+     curl http://{TIDBIP}:10080/ddl/history?"start_job_id={id}&limit={number}"
```
Comment (hawkingrei) [inline docs/tidb_http_api.md:585]:
```suggestion
     curl "http://{TIDBIP}:10080/ddl/history?start_job_id={id}&limit={number}"
```

File: docs/tidb_http_api.md [hunk 2]
```diff
@@ -582,7 +582,7 @@ timezone.*
 27. Get count {number} TiDB DDL job history information, start with job {id}
 
      ```shell
-     curl http://{TIDBIP}:10080/ddl/history?start_job_id={id}&limit={number}
+     curl "http://{TIDBIP}:10080/ddl/history?start_job_id={id}&limit={number}"
```
Comment (zimulala) [inline docs/tidb_http_api.md:585]: There are a lot of uses of "&" in this file so we'd better unify.

Comment (shawn0915) [reply docs/tidb_http_api.md:585]: @hawkingrei could you help take a look again?

**Review by hawkingrei — APPROVED** [review]

File: docs/tidb_http_api.md (hunk 2 above)
Comment (zimulala) [reply docs/tidb_http_api.md:585]: Other urls can be changed together? It's not very uniform

File: docs/tidb_http_api.md [hunk 3]
```diff
@@ -58,11 +58,11 @@
 4. Get the table/index of hot regions
 
     ```shell
-    curl http://{TiDBIP}:10080/regions/hot
```
Comment (zimulala) [inline docs/tidb_http_api.md:61]: why do we need to remove `curl`?

**Review by zimulala — APPROVED** [review]
LGTM

**Pull request merged**